scripts/migrations/11-runtime-cost-adjustment.py

```diff
@@ -0,0 +1,158 @@
+        "create_account_cost": {
+          "send_sir": 99607375000,
+          "send_not_sir": 99607375000,
+          "execution": 99607375000
+        },
+        "deploy_contract_cost": {
+          "send_sir": 184765750000,
+          "send_not_sir": 184765750000,
+          "execution": 184765750000
+        },
+        "deploy_contract_cost_per_byte": {
+          "send_sir": 6812999,
+          "send_not_sir": 6812999,
+          "execution": 6812999
+        },
+        "function_call_cost": {
+          "send_sir": 2319861500000,
+          "send_not_sir": 2319861500000,
+          "execution": 2319861500000
+        },
+        "function_call_cost_per_byte": {
+          "send_sir": 2235934,
+          "send_not_sir": 2235934,
+          "execution": 2235934
+        },
+        "transfer_cost": {
+          "send_sir": 115123062500,
+          "send_not_sir": 115123062500,
+          "execution": 115123062500
+        },
+        "stake_cost": {
+          "send_sir": 141715687500,
+          "send_not_sir": 141715687500,
+          "execution": 102217625000
+        },
+        "add_key_cost": {
+          "full_access_cost": {
+            "send_sir": 101765125000,
+            "send_not_sir": 101765125000,
+            "execution": 101765125000
+          },
+          "function_call_cost": {
+            "send_sir": 102217625000,
+            "send_not_sir": 102217625000,
+            "execution": 102217625000
+          },
+          "function_call_cost_per_byte": {
+            "send_sir": 19715392000000,
+            "send_not_sir": 19715392000000,
+            "execution": 19715392000000
+          }
+        },
+        "delete_key_cost": {
+          "send_sir": 94946625000,
+          "send_not_sir": 94946625000,
+          "execution": 94946625000
+        },
+        "delete_account_cost": {
+          "send_sir": 147489000000,
+          "send_not_sir": 147489000000,
+          "execution": 147489000000
+        }
+      },
+      "storage_usage_config": {
+        "num_bytes_account": 100,
+        "num_extra_bytes_record": 40
+      },
+      "burnt_gas_reward": [
+        3,
+        10
+      ]
+}
+config['runtime_config']['wasm_config']['ext_costs'] = {
+        "base": 265261758,
+        "read_memory_base": 2584050225,
+        "read_memory_byte": 3801396,
+        "write_memory_base": 2780731725,
+        "write_memory_byte": 2723859,
+        "read_register_base": 2493624561,
+        "read_register_byte": 98622,
+        "write_register_base": 2840975211,
+        "write_register_byte": 3801645,
+        "utf8_decoding_base": 3110963061,
+        "utf8_decoding_byte": 289342653,
+        "utf16_decoding_base": 3593689800,
+        "utf16_decoding_byte": 167519322,
+        "sha256_base": 4530046500,
+        "sha256_byte": 24116301,
+        "keccak256_base": 5867223186,
+        "keccak256_byte": 21469644,
+        "keccak512_base": 5798128650,
+        "keccak512_byte": 36651981,
+        "log_base": 0,
```

Comment:
It looks like these were not updated using https://github.com/nearprotocol/nearcore/pull/2621 . Log should become non-zero.

Reply:
Right, they were just hardcoded to be zeros, will fix.